src/main/java/de/danielbechler/diff/differ/PrimitiveArrayDifferFactory.java

```diff
@@ -0,0 +1,11 @@
+package de.danielbechler.diff.differ;
+
+import de.danielbechler.diff.NodeQueryService;
+
+public class PrimitiveArrayDifferFactory implements DifferFactory {

```

Comment:
I only see a method for registering a DifferFactory, not a Differ. I didn’t want to hard-code the PrimitiveArrayDiffer in ObjectDifferBuilder as I understood that you are working on another implementation for array comparison. It should then be the choice of the user which implementation is applicable for him.

 

Von: Daniel Bechler [mailto:notifications@github.com] 
Gesendet: Freitag, 2. Januar 2015 23:22
An: SQiShER/java-object-diff
Cc: Wolfgang-Winter
Betreff: Re: [java-object-diff] PrimitiveArrayDiffer/Field properties (#123)

 

In src/main/java/de/danielbechler/diff/differ/PrimitiveArrayDifferFactory.java <https://github.com/SQiShER/java-object-diff/pull/123#discussion-diff-22425004> :

> @@ -0,0 +1,11 @@
> +package de.danielbechler.diff.differ;
> +
> +import de.danielbechler.diff.NodeQueryService;
> +
> +public class PrimitiveArrayDifferFactory implements DifferFactory {

No need for the factory when you just add the Differ in the ObjectDifferBuilder directly.

—
Reply to this email directly or view it on GitHub <https://github.com/SQiShER/java-object-diff/pull/123/files#r22425004> .  <https://github.com/notifications/beacon/AJovAMfH7weF2b6vf8N8hPHsTwsM6vmgks5ndxGZgaJpZM4DFgj0.gif>